rest_framework/pagination.py

```diff
@@ -715,6 +708,19 @@ def get_ordering(self, request, queryset, view):
             return (ordering,)
         return tuple(ordering)
 
+    def decode_cursor(self, request):
+        # Determine if we have a cursor, and if so then decode it.
+        encoded = request.query_params.get(self.cursor_query_param)
+        if encoded is not None:
+            cursor = _decode_cursor(encoded)
+            if cursor is None:
+                raise NotFound(self.invalid_cursor_message)
+            return cursor
+
+    def encode_cursor(self, cursor):
+        encoded = _encode_cursor(cursor)
```

Comment:
If we *do* choose to have these hooks, would it make sense to remove this extra level of indirection and include the `_encode_cursor` logic directly in the method? (similarly for the decode case) Thoughts?